package.json

```diff
@@ -128,7 +129,7 @@
     "7zip": "0.0.6",
     "@playwright/test": "1.26.0",
     "@types/applicationinsights": "0.20.0",
-    "@types/chart.js": "2.9.4",
+    "@types/chart.js": "2.9.37",
```

Comment:
It looks like the chart.js added types to the main package with v3, so you shouldn't need this anymore.